wagtailio/core/blocks.py

```diff
@@ -179,3 +181,196 @@ class HomePageBlock(blocks.StreamBlock):
 
     class Meta:
         template = "core/blocks/home_page_block.html"
+
+
+class CTABlock(blocks.StructBlock):
+    cta_text = blocks.CharBlock(label="CTA text", max_length=255)
+    cta_page = blocks.PageChooserBlock(label="CTA page", required=False)
+    cta_url = blocks.URLBlock(label="CTA URL", required=False)
+
+    def clean(self, value):
+        struct_value = super(PageOrExternalLinkBlock, self).clean(value)
```

Comment:
nitpick:
```suggestion
        struct_value = super().clean(value)
```

the curent syntax is from the Python 2 days

Reply:
Thanks, this has been fixed